fetch-node.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var wrapFetchForNode = function (fetch) {
+    // Support schemaless URIs on the server for parity with the browser.
+    // https://github.com/matthew-andrews/isomorphic-fetch/pull/10
+    return function (url, options) {
+        if (/^\/\//.test(url)) {
+            url = 'https:' + url;
+        }
+        return fetch.call(undefined, url, options);
+    };
+};
+
+module.exports = function (context) {
+    // Use .bind() as a side effect, which is that it clones. That way,
+    // even if we set `Promise`, we have no side effects on `node-fetch`.
+    var fetch = require('node-fetch').bind(null);
```

Comment:
This is a neat trick I didn't know about. Just so I'm clear what's going on, this is to make it possible to have multiple fetches with potentially different `Promise` implementations, right?